### PR TITLE
[FEATURE] Document new init command

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Reference/GuidesXml.rst [hunk 1]
```diff
@@ -9,19 +9,33 @@ Configuration of the rendering - guides.xml
...
-
 ..  literalinclude:: _guides-simple.xml
     :caption: Documentation/guides.xml
 
+..  contents:: Table of Contents:
+    :local:
+
+..  _guides-xml-init:
+
+Init new guides.xml
```
Comment (jaapio) [inline Documentation/Reference/GuidesXml.rst:20]: I would reword this, what is guides.xml? 

`initialize new documentation project`

Comment (linawolf) [reply Documentation/Reference/GuidesXml.rst:20]: We are on the page, which describes what the guides.xml does here. Therefore I think in this context it is ok
